kogito-editors-js/packages/boxed-expression-component/src/components/DecisionTableExpression/HitPolicySelector.tsx

```diff
@@ -59,6 +59,7 @@ export const HitPolicySelector: React.FunctionComponent<HitPolicySelectorProps>
     (event: React.MouseEvent<Element, MouseEvent>, itemId: string) => {
       const updatedHitPolicy = itemId as HitPolicy;
       const hitPolicySupportsAggregation = _.includes(BUILT_IN_AGGREGATION_AVAILABILITY, updatedHitPolicy);
+      window.beeApi?.notifyUserAction();
```

Comment:
@danielzhe Just a note for a future improvement. In my understanding, directly calling a global function in a component should be avoided. I used the same approach in the `import-java-component` I implemented, and how I'm changing that approach in favor of wrapping that function inside an interface and passing it as prop. So, no need to change this now but keep this in mind for future improvement.
//cc @ljmotta @tiagobento @vpellegrino